ocaml/xapi/cli_operations.ml

```diff
@@ -4798,15 +4798,20 @@ let update_introduce printer rpc session_id params =
   let uuid = Client.Pool_update.get_uuid ~rpc ~session_id ~self:update in
   printer (Cli_printer.PList [uuid])
 
+let livepatch_status_to_string state =
+  match state with
```

Comment:
I assume this match is exhaustive but because it is matching polymorphic variants this is not obvious.